tests/modules/individuals/test_merge.py

```diff
@@ -24,13 +24,13 @@ def test_merge(db, flask_app_client, researcher_1, request, test_root):
     sighting_data = {
         'encounters': [
             {
-                'locationId': str(uuid.uuid4),
+                'verbatimLocality': str(uuid.uuid4),
```

Comment:
Huh, why? I thought verbatimLocality was any old string, why a UUID str?

Reply:
because:
1. it _is_ any old string, thus suffices as a value.  :grin: 
2. the old test (using a random `locationId`) now fails cuz we are actually validating that!

really, i _suspect_ we can let encounters be empty here -- but i wasnt sure!  so i added something.  it really doesnt matter to the test -- especially if we could just pass `encounters: [ {}, {}, {} ]` -- the goal here is to make sure the encounters end up the right place post-merge.  (not what is _in_ the encounters.)